app.py

```diff
@@ -28,12 +38,43 @@ def sign_up_page():
 
 @app.route('/sign_up', methods=['POST'])
 def process_sign_up():
-    if sign_up():
-        return "process completed Eden", 201
-    return "failed to create a new user", 401
+    if process_sign_up():
+        return render_template('signup_completed.html'), 200
+    return render_template('signup_railed.html'), 400
+
+
+@app.route('/login')
+def return_login_page():
+    return render_template('login_form.html')
+
 
+@app.route('/login', methods=['POST'])
+def process_login():
+    if login_verification():
+        return render_template('login_completed.html'), 200
+    return render_template('login_failed.html'), 401
 
-def sign_up():
+
+def login_verification():
+    pass_text = request.form['pass']
+    username_text = request.form['name']
+    return existing_user(username_text, pass_text)
+
+
+def existing_user(str_username, str_pass):  # work here
+    conn = sqlite3.connect('data_base.db')
+    curs = conn.cursor()
+    # curs.execute('SELECT * FROM users WHERE Username = ? AND Password = ?',
+                 #(str_username, str_pass)).fetchall()
+
+    curs.execute('''SELECT * FROM users WHERE Username = ? AND Password = ?;''',
```

Comment:
Maybe wrap it inside a function in a different class? separate your SQL implementation from the views in flask. That way if you want to change something in the software, you only need to change it in one place.

Reply:
will do